src/universal/modules/email/components/SummaryEmail/SummaryEmail.js

```diff
@@ -104,27 +104,12 @@ const SummaryEmail = (props) => {
     teamDashUrl
   } = props;
   const {agendaItemsCompleted, invitees, createdAt, meetingNumber, teamName} = meeting;
-  const membersSansOutcomes = [];
-  const membersWithOutcomes = [];
-  let presentMemberCount = 0;
-  let actionCount = 0;
-  let projectCount = 0;
-
-  for (let i = 0; i < invitees.length; i++) {
-    const invitee = invitees[i];
-    if (invitee.present) {
-      presentMemberCount++;
-    }
-    const projLen = invitee.projects.length;
-    const actionLen = invitee.actions.length;
-    actionCount += actionLen;
-    projectCount += projLen;
-    const arr = (!projLen && !actionLen) ? membersSansOutcomes : membersWithOutcomes;
-    arr.push(invitee);
-  }
+  const membersSansOutcomes = invitees.filter((invitee) => invitee.projects.length === 0);
```

Comment:
very true, i stopped about halfway through because it's part of a bigger conversation. Personally, i think we should rename "Project" to "Task" to encourage more card creation. Projects feel big (eg Microsoft Project, Project Management Professional, etc). Task feels like a todo.